CMakeLists.txt

```diff
@@ -1646,6 +1631,46 @@ if(WOLFSSL_CRYPTOCB)
     list(APPEND WOLFSSL_DEFINITIONS "-DWOLF_CRYPTO_CB")
 endif()
 
+if(WOLFSSL_OCSPSTAPLING)
+    set(WOLFSSL_CERTIFICATE_STATUS_REQUEST "yes")
```

Comment:
Do we need `WOLFSSL_CERTIFICATE_STATUS_REQUEST` and `WOLFSSL_CERTIFICATE_STATUS_REQUEST_V2`? Seems simpler to avoid this intermediate variable and just do

```
if(WOLFSSL_OCSPSTAPLING)
    set(BUILD_OCSP_STAPLING "yes" PARENT_SCOPE)
endif()
```

in functions.cmake, unless I'm missing something.

Reply:
Done. I had considered it but was ambivalent.